bigcache.go

```diff
@@ -44,7 +50,7 @@ func NewBigCache(config Config) (*BigCache, error) {
 func newBigCache(config Config, clock clock) (*BigCache, error) {
 
 	if !isPowerOfTwo(config.Shards) {
-		return nil, fmt.Errorf("Shards number must be power of two")
+		return nil, fmt.Errorf("shards number must be power of two")
```

Comment:
Do not change errors, please.

Reply:
@cristaloleg i changed this because of linting errors. If you don't want me to run linting on the repo i can revert the changes. More information about the error `ST1005: error strings should not be capitalized (stylecheck)` from https://staticcheck.io/docs/checks

Reply:
@cristaloleg any more questions on the change here

Reply:
Yep, I know, I don't like linter warnings, but changing this variable will change api. In the near future we will release a new major version where all such warnings will be resolved, but not currently. Thank you.